### PR TITLE
Update rootkie.md for PPP

### PR DESCRIPTION
**Pull request by rootkie**
update my PPP

### CODE REVIEW

File: docs/team/rootkie.md [hunk 1]
```diff
@@ -3,44 +3,58 @@ layout: page
...
+  * Reported 11 bugs for [PED](https://github.com/rootkie/ped/issues) including non-trivial logic bugs. 
+  
 * **Tools**:
-  * Integrated a third party library (Natty) to the project ([\#42]())
-  * Integrated a new Github plugin (CircleCI) to the team repo
+  * Setting up CI and CodeCov for project.
 
-* _{you can add/remove categories in the list above}_
+* **Statistics**:
+  * PRs reviewed: [52](https://github.com/AY2122S1-CS2103T-W11-1/tp/pulls?q=is%3Apr+is%3Aclosed+reviewed-by%3A%40me)
```
Comment (rebeccacxy) [inline docs/team/rootkie.md:57]: Faulty link

File: docs/team/rootkie.md [hunk 2]
```diff
@@ -3,44 +3,58 @@ layout: page
 title: Bailin's Project Portfolio Page
 ---
 
-### Project: AddressBook Level 3
+### Project: Fridgy
 
-AddressBook - Level 3 is a desktop address book application used for teaching Software Engineering principles. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
+Fridgy is a desktop a Food Inventory Manager that makes it easy to reduce food waste by keeping track of what you have and when it expires. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 14 kLoC.
```
Comment (pattan9955) [inline docs/team/rootkie.md]: Typo here, it should be "Fridgy is a desktop Food Inventory Manager...."

File: docs/team/rootkie.md [hunk 3]
```diff
@@ -3,44 +3,58 @@ layout: page
...
 
 Given below are my contributions to the project.
 
-* **New Feature**: Added the ability to undo/redo previous commands.
-  * What it does: allows the user to undo all previous commands one at a time. Preceding undo commands can be reversed by using the redo command.
-  * Justification: This feature improves the product significantly because a user can make mistakes in commands and the app should provide a convenient way to rectify them.
-  * Highlights: This enhancement affects existing commands and commands to be added in future. It required an in-depth analysis of design alternatives. The implementation too was challenging as it required changes to existing commands.
-  * Credits: *{mention here if you reused any code/ideas from elsewhere or if a third-party library is heavily used in the feature so that a reader can make a more accurate judgement of how much effort went into the feature}*
+* **New Feature**: Added the ability view any ingredient / recipe on the main display window.
```
Comment (pattan9955) [inline docs/team/rootkie.md:12]: Should be "the ability to view...."

File: docs/team/rootkie.md (hunk 1 above)
Comment (rootkie) [reply docs/team/rootkie.md]: Ah good catch 👍

**Review by alissayarmantho — APPROVED** [review]
LGTM

**Pull request merged**